### PR TITLE
Translate v4 document to zhCN

### PR DESCRIPTION
**Pull request by mikefoxnobel**
@dhmlau @agnes512 @nabdelgadir 
Refer to https://github.com/strongloop/loopback.io/pull/887 this pull request.
Sorry for that hard reset auto closed the pull request.

All the review comments in that pull request is applied already.

### CODE REVIEW

**Comment by agnes512** [comment]
LGTM! 👏 
one last step, could you squash the commits to one? steps are in the comment: https://github.com/strongloop/loopback.io/pull/887#issuecomment-534539609
I will merge the PR once it is squashed. Thank you!!

File: pages/zh/lb4/Controllers.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
+---
+lang: zh
+title: '控制器（Controllers）'
+keywords: LoopBack 4.0, LoopBack 4
```
Comment (nabdelgadir) [inline pages/zh/lb4/Controllers.md:4]: please add `layout: translation` for each page that's been translated

File: _data/sidebars/zh_lb4_sidebar.yml [hunk 2]
```diff
@@ -0,0 +1,821 @@
+## definition of `menuitem`===========
```
Comment (nabdelgadir) [inline _data/sidebars/zh_lb4_sidebar.yml:1]: please add this sidebar to [`_config.yml`](https://github.com/mikefoxnobel/loopback.io/blob/gh-pages-zhCN/_config.yml)

**Comment by mikefoxnobel** [comment]
@agnes512 I squashed it, seems it got problem again.
My "Sourcetree" git client will auto generate a long long commit message during merge.

**Comment by agnes512** [comment]
yes, during merging it would generates some messages but once you squash successfully, it will only have one commit message left. 
For example, if you have conflicts during merging,
- fix
- `git add .`
- `git rebase --continue`. (you don't need to `git commit` here)
- then `:wq` the generated commit message if there is any. 

Repeat  these steps till it success. Since you have 9 commits now, you might need to repeat several times to resolver the conflicts between each commi.

Once you're done, if the one left commit message got changed ( this happens sometimes), you just need to `git commit --amend` to edit your commit message.
Last step: `git push origin +YOURBRANCHNAME`

Let me know if this works.

**Comment by agnes512** [comment]
Hi @mikefoxnobel I just squashed and edited the commit message for you. Hope this saves you some time :smile:!
Feel free to change the commit message. I will merge it once you think it looks good to you. Thank you again!

**Comment by mikefoxnobel** [comment]
@agnes512 thanks very much.
Please merge it.

**Review by nabdelgadir — COMMENTED** [review]
LGTM 👏

**Pull request merged**